client/ayon_resolve/api/pipeline.py

```diff
@@ -156,6 +156,10 @@ def ls():
             continue
         data = json.loads(data)
 
+        # treat data as container
+        if data.get("load"):
+            data = data["load"]
```

Comment:
Not sure about this - why do we need this. I feel this comment is also lacking as to for what use case this is. Might be good to elaborate.

Reply:
I hope it makes more sense now 64071b6cafaf7ec724ffea01890937ed6b8fa206